src/server/formatting.rs

```diff
@@ -1,10 +1,14 @@
+use std::{fs::File, io::Read};
+
 use tower_lsp::lsp_types::{Position, Range, Registration, TextEdit, Unregistration};
 use typst::syntax::Source;
+use typstfmt_lib::Config as FmtConfig;
 
 use super::TypstServer;
 
 const FORMATTING_REGISTRATION_ID: &str = "formatting";
 const DOCUMENT_FORMATTING_METHOD_ID: &str = "textDocument/formatting";
+const CONFIG_PATH: &str = "typstfmt-config.toml";
```

Comment:
Might it be worth making this the default, but allowing it to be configured?

Reply:
That's how [typstfmt](https://github.com/astrale-sharp/typstfmt/blob/master/src/main.rs#L79) does it, so I figured we'd just follow along. Your call though, I bet I can work this out ;)

Reply:
Or would you prefer passing the config as a options for typst-lsp? Doesn't feel totally right b/c typstfmt explicitely reads this file... still, your call :)

Reply:
what about a transparent option like `rust-analyzer.rustfmt.extraArgs`? Similarly, we can put all of their options in `typst-lsp.typstfmt.options`. Then we can write:

```json
{
  "typst-lsp.typstfmt.extraArgs": ["--config", "name"]
}
```

Reply:
See my answer below, `typst-lsp` is not using the binary.

Reply:
I haven't actually used CLI `typstfmt`, so I didn't realize the config file isn't itself configurable. Given that, this is fine, though ideally `typstfmt` would export the filename so we don't need to bake it in. That said, I'm treating the formatter integration as explicitly experimental for now, so this is fine, too.